src/templates/finops-hub/modules/dataFactory.bicep

```
@@ -1027,7 +1084,7 @@ resource pipeline_InitializeHub 'Microsoft.DataFactory/factories/pipelines@2018-
                 commandTimeout: '00:20:00'
               }
               linkedServiceName: {
-                referenceName: linkedService_dataExplorer.name
+                referenceName: dataExplorerName
```

Doesn't this go against our pattern of using the resource to get the name? Referencing the actual resource name will protect us from any possible name changes.
```suggestion
                referenceName: linkedService_dataExplorer.name
```

---

Doesn't work if the resource is optional as the object will evaluate as null if we don't deploy ADX.  This causes some timing challenges too.